### ISSUE TITLE
Bump minimist and mkdirp

### ISSUE THREAD

**Issue by jeffpaul** [cedaro/node-wp-i18n#25]
In trying to trace down some Dependabot alerts, updating the following two lines would help unblock some downstream packages to similarly update:

https://github.com/cedaro/node-wp-i18n/blob/4633a5332b643e2e74de4ece96ec1c0232ed4914/package.json#L27
Potentially bump to [latest](https://github.com/substack/minimist/releases):
```
"minimist": "^1.2.5",
```

https://github.com/cedaro/node-wp-i18n/blob/4633a5332b643e2e74de4ece96ec1c0232ed4914/package.json#L28
Potentially bump to [latest](https://github.com/isaacs/node-mkdirp/releases):
```
"mkdirp": "^v1.0.4",
```
...or...
```
"mkdirp": "^v0.5.5",
```

Will need to ensure those don't break any functionality, but will allow downstream packages to similarly account for the updates to `minimist` (which is also included in `mkdirp`).

**Comment by bradyvercher** [comment]
@jeffpaul I think this should be good to go with the latest commit. I'll try to get it released after tracking down another issue.

**Comment by jeffpaul** [comment]
@bradyvercher checking back on releasing to help allow downstream projects update `node-wp-i18n` and alleviate a known, albeit low severity, issue with `minimist`.

Related CVE details: https://github.com/advisories/GHSA-vh95-rmgr-6w4m

**Comment by bradyvercher** [comment]
@jeffpaul Thanks for the nudge. This should be good to go in 1.2.4.
